apps/openmw/engine.cpp

```diff
@@ -140,6 +138,7 @@ void OMW::Engine::frame(float frametime)
             mEnvironment.getMechanicsManager()->update(frametime,
                 guiActive);
         }
+        mEnvironment.getWorld()->markCellAsUnchanged();
```

Comment:
I went ahead and looked at it now, and used `isInCell()` instead, which seems to work. I undid the change to engine.cpp. I pushed it to the branch but GitHub seems to be slow to update right now.